tests/daemon-startup.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+#
+# Check that messages that should be sent on startup are actually sent.
+
+source sh-helper
+assert cargo jq
+
+echo " This test takes about 10s"
+
+output=$(tmpfile daemon-startup-output)
+log=$(tmpfile daemon-startup-log)
+inifile=$(tmpfile daemon-startup-ini)
+
+# The startup message will fire (should fire) and then sonar will exit.  If they don't fire, sonar
+# will hang because the cadence is very, very slow (8 days).  For that reason, kill it if it is
+# still there after a few seconds.
+#
+# With very low probability, the timed messages will fire at the same time as the startup message.
+# Over time, this probability approaches zero.
```

Comment:
That's a clunky way of saying that the test may intermittently fail but it will be obvious that it is an intermittent.  Should probably reword.